lib/wallet/api/http/Cardano/Wallet/Api/Types.hs

```diff
@@ -1216,7 +1216,6 @@ data ApiNetworkParameters = ApiNetworkParameters
     , activeSlotCoefficient :: !(Quantity "percent" Double)
     , decentralizationLevel :: !(Quantity "percent" Percentage)
     , desiredPoolNumber :: !Word16
-    , maximumTokenBundleSize :: !(Quantity "byte" Natural)
```

Comment:
It's tempting to remove this field, and I think it should be fine, but requires a discussion

Reply:
@jonathanknowles thoughts?

Reply:
I've now kept `maximumTokenBundleSize` and duplicated the `TokenBundleMaxSize` definition.